masterlist.yaml

```
@@ -26903,6 +26903,31 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0x76F44BFE
         util: 'SSEEdit v4.0.4c'
+        
+  - name: 'Solitude Docks.esp'
```

There's also some extra whitespace that I missed earlier:
```suggestion
        util: 'SSEEdit v4.0.4c'

  - name: 'Solitude Docks.esp'
```